x/wasm/internal/keeper/handler_plugin.go

```diff
@@ -99,12 +99,12 @@ func EncodeBankMsg(sender sdk.AccAddress, msg *wasmTypes.BankMsg) ([]sdk.Msg, er
 	if err != nil {
 		return nil, err
 	}
-	sdkMsg := bank.MsgSend{
-		FromAddress: fromAddr,
-		ToAddress:   toAddr,
+	sdkMsg := banktypes.MsgSend{
+		FromAddress: fromAddr.String(),
```

Comment:
Same here. And anywhere we see `sdk.AccAddressFromBech32()`. Just remove those calls and the `.String()` and use the address directly from the contract (the receiving module must parse and validate it)